web/dist/index.html

```diff
@@ -233,6 +233,17 @@ <h4 class="modal-title" id="myModalLabel">Connect to remote node...</h4>
                           </div>
                         </div>
                       </div>
+                      <div id="bandwidthInputDiv">
+                        <label for="bandwidthInput" class="col-md-2 control-label"
+                          >bandwidth:</label
+                        >
+                        <div class="col-md-3">
+                          <div class="input-group">
+                            <input type="text" class="form-control" id="bandwidthInput" />
```

Comment:
Don't know if you noticed, but I added [`func Bandwidths() []Bandwidth`](https://pkg.go.dev/github.com/la5nta/wl2k-go@v0.9.2/transport/ardop#Bandwidths). This function returns all valid ARDOP ARQ bandwidths.

If we add an API endpoint returning valid bandwidth options per transport, we could make this a dropdown to help the user navigate the different bandwidth options.

What do you think?

Reply:
It's an extra bit of work, but yes, it's worth the effort. I'll make that change.

Reply:
Done.